src/main/kotlin/com/project/goms/domain/outing/entity/OutingStatus.kt

```diff
@@ -0,0 +1,9 @@
+package com.project.goms.domain.outing.entity
+
+enum class OutingStatus {
```

Comment:
코드 단에서 사용할때 OutingStatus. 하고 참조를 하기 때문에
외출 상태라는 말을 이미 앞에서 붙혔는데 OutingStatus.OUTING_AVAILABLE(외출상태.외출_가능)이라는 말 보다는 OutingStatus.AVAILABLE(외출상태.가능)으로 수정해서 Outing(외출)이라는 중복어를 제거하면 좋을거 같은데 어떻게 생각하실까요?
```suggestion
enum class OutingStatus {
    AVAILABLE,
    PUBLIC_HOLIDAY,
    UNAVAILABLE
}

```